services/core/shared/core/compat/sdk_v5/blocks.js

```diff
@@ -300,11 +311,11 @@ const getBlocks = async params => {
 	}
 
 	try {
-		if (params.id) {
+		if (params.ids) {
+			blocks.data = await getBlocksByIDs(params.ids);
+		} else if (params.id) {
 			blocks.data = await getBlockByID(params.id);
 			if ('offset' in params && params.limit) blocks.data = blocks.data.slice(params.offset, params.offset + params.limit);
```

Comment:
With the updated implementation of the `isQueryFromIndex` method, the following check may not be necessary.
```suggestion
```